pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Worker.java

```diff
@@ -129,4 +129,19 @@ public WorkerInfo getClusterLeader() {
     public void rebalance() {
         worker.rebalance(uri.getRequestUri(), clientAppId());
     }
+
+    @GET
+    @ApiOperation(
+            value = "Checks if this node is the leader and is ready to service requests",
+            response = String.class
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 401, message = "The requester is not authenticated"),
+            @ApiResponse(code = 503, message = "Worker service is not running")
+    })
+    @Path("/cluster/leaderready")
```

Comment:
I think the url path would be more clean if it is /cluster/leader/ready.  We can add more API under "/leader" in future

Reply:
Beside we already have have the url path "/cluster/leader"

Reply:
Changed